.github/scripts/run_tests.sh

```diff
@@ -44,8 +44,9 @@ function run_torch_xla_cpp_tests() {
                "test_aten_xla_tensor_2"
                "test_aten_xla_tensor_3"
                "test_aten_xla_tensor_4"
-               "pjrt_computation_client_test"
-               "ifrt_computation_client_test")
+               "pjrt_computation_client_test")
+               # Disable IFRT test as it currently crashes
+               #"ifrt_computation_client_test")
```

Comment:
Why does it crash? leave a note?

Reply:
Saw it in the PR comment, `//test/cpp:test_xla_sharding` is the failed test right?

Reply:
I guess that's the thing I don't know, it crashed right away for any XLA graph execution. I decided to not digging too much giving no one is using ifrt right now.

Reply:
ifrt_computation_client_test is the only test that we have for `ifrt` and it is crashing. It also crashed on any ifrt execution.